components/cloud-foundry/frontend/src/api/Apps.js

```diff
@@ -7,6 +7,8 @@
     .module('cloud-foundry.api')
     .run(registerApi);
 
+  var BASE_URL = '/pp/v1/proxy/v2/apps';
```

Comment:
In the past we've avoided making changes to 'api' files as they were originally auto-generated. Going forward I don't think there's much harm in re-factoring but we need to bear in mind we might auto-generate these files for v3 of the cf api.